OrbitQt/orbitmainwindow.cpp

```diff
@@ -744,6 +746,23 @@ void OrbitMainWindow::RestoreDefaultTabLayout() {
   UpdateCaptureStateDependentWidgets();
 }
 
+void OrbitMainWindow::CheckAndChangeFunctionHighlight() {
+  if (GOrbitApp->selected_thread_id() != SamplingProfiler::kAllThreadsFakeTid) {
```

Comment:
Why is the check against kAllThreadFakeTid needed?

Reply:
Thanks for pointing out this. I misunderstood the purpose for checking against kAllThreadFakeTid. I thought this will filter out some uninterested thread id. I will read the OrbitGl codes more carefully.